search-app/README.md

```diff
@@ -4,8 +4,27 @@ Provides a public search API for concepts in the CMR.
 
 ## API Documentation
 
-API Docs are located in api_docs.md. They can be generated into the static site by running `lein generate-docs` This takes a while so we will commit the generated file for now so building the uberjar does not take a very long time. You can also generate the documents with a pre-started REPL by running `(load-file "./support/generate_docs.clj")` If you're running the REPL from dev-system you need to correct the path to generate_docs.clj.
+API docs for the search-app project are located in `api_docs.md`. They can be
+generated into the static site by running the following:
+
+```
+$ lein generate-docs
+```
+
+Due to the fact that this generally takes a while on most systems, we have
```

Comment:
I realize this is existing documentation, but I think it is outdated. We run lein with-profile docs generate-docs and the documentation is generated in seconds. I believe we do this as part of the build and we aren't committing the generated file anymore.